lib/ansible_tower_client/job_template.rb

```diff
@@ -27,5 +34,28 @@ def survey_spec
     def self.endpoint
       "job_templates".freeze
     end
+
+    private
+
+    def with_temporary_changes(limit)
+      old_limit = @raw_body['limit']
+      new_limit = limit
+      patch("{ \"limit\": \"#{new_limit}\" }")
+      begin
+        yield
+      rescue => e
+        raise InvalidPost.new(e)
+      ensure
+        patch("{ \"limit\": \"#{old_limit}\" }")
+      end
+    end
+
+    def patch(body)
+      api.patch do |req|
+        req.url url
```

Comment:
If we all follow the same style by manageiq, we should write as `req.url(url)`. I am surprised at how to setup the `url` vs `body` as the syntax is different.